sdo/material_sdo_context.jsonld

```diff
@@ -11,10 +11,50 @@
       "@type": "sdo:Text"
     },
     "description": {
-        "@id": "sdo:name",
-        "@type": "sdo:Text"
+      "@id": "sdo:name",
+      "@type": "sdo:Text"
     },
     "spatialCoverage": "sdo:spatialCoverage",
-    "roles": "sdo:roleName"
+    "roles": "sdo:roleName",
+    "alternateIdentifiers": {
+      "@id": "sdo:identifier",
+      "@type": "sdo:Text"
+    },
+    "relatedIdentifiers": {
+      "@id": "sdo:identifier",
+      "@type": "sdo:Text"
+    },
+    "derivesFrom": {
+      "@id": "sdo:isBasedOn",
+      "@type": "sdo:Thing"
+    },
+    "bearerOfDisease": {
+      "@id": "sdo:diagnosis",
+      "@type": "sdo:MedicalCondition"
+    },
+    "taxonomy": {
+      "@id": "sdo:category",
+      "@type": "sdo:Thing"
+    },
+    "involvedInBiologicalEntity": {
+      "@id": "sdo:potentialAction",
+      "@type": "sdo:Thing"
+    },
+    "characteristics": {
+      "@id": "sdo:additionalProperty",
+      "@type": "sdo:PropertyValue"
+    },
+    "consentInformation": {
+      "@id": "sdo:permissionType",
+      "@type": "sdo:DigitalDocumentPermissionType"
+    },
+    "dates": {
+      "@id": "sdo:temporalCoverage",
+      "@type": "sdo:DigitalDocumentPermissionType"
```

Comment:
Should be one of these data types DateTime, Text or URL. The `Text ` probably is the best option.

Reply:
Thanks for catching this. I believe this was a mixup in the mappings - DATS doesn't declare any `@type` for "dates" I think. So `sdo:Text` would work well.